src/components/Tabs/SettingsTab.tsx

```diff
@@ -139,85 +142,8 @@ export default function SettingsTab(): ReactElement {
-                type="checkbox"
-                disabled={isBackdropDisabled}
-                checked={backdropVisible}
-                onChange={(event) => setBackdropVisible(event.target.checked)}
-              />
-            </div>
-          </SettingsItem>
-          <SettingsItem label="Backdrop" htmlFor={SettingsHtmlIds.BACKDROP_KEY_SELECT}>
-            <SelectionDropdown
-              id={SettingsHtmlIds.BACKDROP_KEY_SELECT}
-              selected={selectedBackdropKey}
-              items={backdropOptions}
-              onChange={(key) => dataset && setBackdropKey(key)}
-              disabled={isBackdropOptionsDisabled}
-            />
-          </SettingsItem>
-          <SettingsItem label="Brightness" htmlFor={SettingsHtmlIds.BACKDROP_BRIGHTNESS_SLIDER}>
-            <div style={{ maxWidth: MAX_SLIDER_WIDTH, width: "100%" }}>
-              <LabeledSlider
-                id={SettingsHtmlIds.BACKDROP_BRIGHTNESS_SLIDER}
-                type="value"
-                minSliderBound={0}
-                maxSliderBound={200}
-                minInputBound={0}
-                maxInputBound={200}
-                value={backdropBrightness}
-                onChange={setBackdropBrightness}
-                marks={[100]}
-                numberFormatter={(value?: number) => `${value}%`}
-                disabled={isBackdropOptionsDisabled}
-              />
-            </div>
-          </SettingsItem>
-
-          <SettingsItem label="Saturation" htmlFor={SettingsHtmlIds.BACKDROP_SATURATION_SLIDER}>
-            <div style={{ maxWidth: MAX_SLIDER_WIDTH, width: "100%" }}>
-              <LabeledSlider
-                id={SettingsHtmlIds.BACKDROP_SATURATION_SLIDER}
-                type="value"
-                minSliderBound={0}
-                maxSliderBound={100}
-                minInputBound={0}
-                maxInputBound={100}
-                value={backdropSaturation}
-                onChange={setBackdropSaturation}
-                marks={[100]}
-                numberFormatter={(value?: number) => `${value}%`}
-                disabled={isBackdropOptionsDisabled}
-              />
-            </div>
-          </SettingsItem>
-          <SettingsItem label="Object opacity" htmlFor={SettingsHtmlIds.OBJECT_OPACITY_SLIDER}>
-            <div style={{ maxWidth: MAX_SLIDER_WIDTH, width: "100%" }}>
-              <LabeledSlider
-                id={SettingsHtmlIds.OBJECT_OPACITY_SLIDER}
-                type="value"
-                disabled={isBackdropOptionsDisabled}
-                minSliderBound={0}
-                maxSliderBound={100}
-                minInputBound={0}
-                maxInputBound={100}
-                value={objectOpacity}
-                onChange={setObjectOpacity}
-                marks={[100]}
-                numberFormatter={(value?: number) => `${value}%`}
-              />
-            </div>
-          </SettingsItem>
-        </SettingsContainer>
-      </CustomCollapse>
```

Comment:
Except for replacing `CustomCollapse` with `CheckboxCollapse`, there were no other changes to this code move.